src/shapes.c

```diff
@@ -116,45 +116,17 @@ void DrawLineV(Vector2 startPos, Vector2 endPos, Color color)
 // Draw a line defining thickness
 void DrawLineEx(Vector2 startPos, Vector2 endPos, float thick, Color color)
 {
-    if (startPos.x > endPos.x)
-    {
-        Vector2 tempPos = startPos;
-        startPos = endPos;
-        endPos = tempPos;
-    }
-
-    float dx = endPos.x - startPos.x;
-    float dy = endPos.y - startPos.y;
-
-    float d = sqrtf(dx*dx + dy*dy);
-    float angle = asinf(dy/d);
+    Vector2 delta = Vector2Subtract(endPos, startPos);
```

Comment:
I tried to avoid `raymath` dependency on the `shapes` module, could you just do the required maths there?

Reply:
I made the changes you requested.